src/Illuminate/Foundation/Console/OptimizeCommand.php

```diff
@@ -96,19 +100,22 @@ protected function compileClasses()
     }
 
     /**
-     * Get the node traverser used by the command.
+     * Get the class preloader used by the command.
      *
-     * @return \ClassPreloader\Parser\NodeTraverser
+     * @return \ClassPreloader\ClassPreloader
      */
-    protected function getTraverser()
```

Comment:
If someone installs 2.x, the optimize command works just fine. If someone installs 3.x, it also works, but we use the new factory class rather than building the traverser ourselves.